### DISCUSSION TITLE
YouTube 标题未覆盖

### DISCUSSION THREAD

**Discussion by yuuuxt** [F9y4ng/GreasyFork-Scripts#103] [问答专区(Question / Answer)] [answered]
YouTube的标题是`<yt-formatted-string></yt-formatted-string>`(打开首页即可), 目前没有被覆盖到(如下图, 没有应用自定义字体):
![image](https://user-images.githubusercontent.com/3153534/177004613-f4404bc3-9f4b-4954-8035-425837f8e888.png)

**Comment by F9y4ng** [comment, marked as answer]
默认情况下，脚本已对中文字体进行热替换，如果你查看中文标题的情况下即可看到标题中文部分已被成功热替换。

对英文字体的渲染在 **需要渲染的网页元素** 中，追加`,#video-title.ytd-rich-grid-media`，保存为站点独享。

**Reply by yuuuxt** [reply]
> 默认情况下，脚本已对中文字体进行热替换，如果你查看中文标题的情况下即可看到标题中文部分已被成功热替换。
> 
> 对英文字体的渲染在 **需要渲染的网页元素** 中，追加`,#video-title.ytd-rich-grid-media`，保存为站点独享。

首页问题解决;

以及具体视频页的右侧相关视频列表也是缺失选择, 有同样的问题的, 也建议补充一下 :)

**Reply by F9y4ng** [reply]
同样的方法，找到对应的未渲染的元素，继续用`,`+`样式选择器` 追加到 **需要渲染的网页元素** 里。

**Reply by F9y4ng** [reply]
全站效果：**需要渲染的网页元素** 追加 `,#video-title[class*='ytd-'],ytd-compact-radio-renderer[class*='-'] #video-title[class*='ytd-']`
